zone/client.cpp

```diff
@@ -3807,6 +3882,20 @@ float Client::CalcPriceMod(Mob* other, bool reverse)
 	return chaformula; //Returns 1.10, expensive stuff!
 }
 
+float Client::CalcPriceMod(Mob* other, bool reverse)
+{
+	float price_mod;
+
+	if (RuleB(Merchant, UseClassicPriceMod)) {
+		price_mod = CalcClassicPriceMod(other, reverse);
+	}
+	else {
+		price_mod = CalcNewPriceMod(other, reverse);
+	}
+
+	return price_mod;
```

Comment:
```cpp
return RuleB(Merchant, UseClassicPriceMod) ? CalcClassicPriceMod(other, reverse) : CalcPriceMod(other, reverse);
```

Reply:
Can we leave this alone?  I like the clarity and the funtion is tiny.  I don't see any need to shorten it.

Reply:
That's fine.We can at least have the `price_mod` have a default value.
```cpp
float Client::CalcPriceMod(Mob* other, bool reverse)
{
	float price_mod = CalcNewPriceMod(other, reverse);

	if (RuleB(Merchant, UseClassicPriceMod)) {
		price_mod = CalcClassicPriceMod(other, reverse);
	}

	return price_mod;
}
```